src/Activators/FileActivator.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Nwidart\Modules\Activators;
+
+use Illuminate\Cache\CacheManager;
+use Illuminate\Config\Repository as Config;
+use Illuminate\Container\Container;
+use Illuminate\Filesystem\Filesystem;
+use Nwidart\Modules\Contracts\ActivatorInterface;
+use Nwidart\Modules\Module;
+
+class FileActivator implements ActivatorInterface
+{
+	/**
+	 * Laravel cache instance
+	 * 
+	 * @var CacheManager
+	 */
+	protected $cache;
+
+	/**
+	 * Laravel Filesystem instance
+	 * 
+	 * @var Filesystem
+	 */
+	protected $files;
+
+	/**
+	 * Laravel config instance
+	 * @var Config
+	 */
+	protected $config;
+
+	/**
+	 * @var string
+	 */
+	protected $cacheKey;
+
+	/**
+	 * @var string
+	 */
+	protected $cacheLifetime;
+
+	/**
+	 * Array of modules activation statuses
+	 * 
+	 * @var array
+	 */
+	protected $installed;
+
+	/**
+	 * File used to store activation statuses
+	 * 
+	 * @var string
+	 */
+	protected $fileInstalled;
+
+	public function __construct(Container $app)
+	{
+		$this->cache = $app['cache'];
+		$this->files = $app['files'];
+		$this->config = $app['config'];
+		$this->fileInstalled = $this->config('file');
```

Comment:
`fileInstalled` seems weird. Also the fact that there's a difference between the config key and the class field means there's something off with the naming. 
It should be the same in config key and class field. 

In order to chose a good name, What is this file? What does it contain? 
It looks like it contains a list of all active modules? 
If so we should use that keyword and avoid "install"